docs/usage/chips/chip_presence_detection.md

```diff
@@ -24,8 +24,8 @@ This `chip` shows you the actual presence in your home. Shows residents and gues
 - type: 'custom:button-card'
   template: chip_presence_detection
   variables:
-    ulm_chip_presence_counter_residents: sensor.my_residents_counter
-    ulm_chip_presence_counter_guests: sensor.my_guests_counter
+    ulm_chip_presence_counter_residents: sensor.people_home
+    ulm_chip_presence_counter_guests: sensor.people_home
```

Comment:
That should probably be rather `guests_home`? But as a example it is also ok, I think ;-)

Reply:
Oh yes my bad. You could change it or you want me to do another push ?

Reply:
When you can do another push it would be great :-)